pyop2/base.py

```diff
@@ -4100,10 +4136,11 @@ def num_flops(self):
             size = iterset.exec_size
         if self.is_indirect and iterset._extruded:
             region = self.iteration_region
+            layers = np.mean(iterset.layers_array[:, 1] - iterset.layers_array[:, 0])
```

Comment:
Yes, because the total number of cells in the mesh is sum(layers), which is equal to sum(layers)/ncell * ncell.